baseapp/baseapp_test.go

```diff
@@ -638,6 +639,318 @@ func TestValidatorChange(t *testing.T) {
 
 //----------------------------------------
 
+// Use burn and send msg types to test multiple msgs in one tx
+type testBurnMsg struct {
```

Comment:
Will now need a `GetMemo()` function (which can return an empty string); rebase against develop (https://github.com/cosmos/cosmos-sdk/pull/1280 has since been merged).